lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -4638,7 +4638,7 @@ unsigned LoopVectorizationCostModel::selectInterleaveCount(bool OptForSize,
     return 1;
 
   // We used the distance for the interleave count.
-  if (Legal->getMaxSafeDepDistBytes() != std::numeric_limits<unsigned>::max()) {
+  if (Legal->getMaxSafeDepDistBytes() != std::numeric_limits<unsigned>::max())
```

Comment:
Loop vectorization is completely disabled in DXC. That would be why this didn't fail. Might as well get it right anyway though.